packages/kolibri/components/pages/AppBarPage/internal/SideNav.vue

```diff
@@ -434,6 +452,13 @@
           }
         });
       },
+      startTourTrigger(newVal) {
+        if (newVal) {
+          this.$nextTick(() => {
+            this.startTour('SideNavigation');
```

Comment:
Instead of adding a new property for the `SideNav` and setting it from the `AppBar` and `AppBarPage` components, this line could be moved to just after line 451, right after `this.focusFirstEl();` which might help simplify the overall structure a little.